pkg/blackduck/ctl_blackduck_test.go

```diff
@@ -51,7 +51,7 @@ func TestNewBlackduckCtl(t *testing.T) {
 		LivenessProbes:                        false,
 		ScanType:                              "",
 		PersistentStorage:                     false,
-		PVCJSONSlice:                          []string{},
+		PVCFile:                               "",
```

Comment:
All of these are actually FilePaths, right?

Reply:
Yes, do you think the variables should be renamed to PVCFilePath, etc... ?

Reply:
I think so, for readability purposes :D